google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java

```diff
@@ -283,6 +304,10 @@ private void hideError() {
     errorPanel.setVisible(false);
   }
 
+  private void pingMetrics(String metricsEvent) {
```

Comment:
Can this be static? Also, wouldn't it be more appropriate to put this in the `UsageTrackerProvider` class and static import it here for usage? That way you could re-use it elsewhere to reduce boilerplate.